hack/test-end-to-end.sh

```diff
@@ -328,6 +313,9 @@ wait_for_app "test"
 
 # ensure the router is started
 # TODO: simplify when #4702 is fixed upstream
+echo "[INFO] Back to 'master' context with 'admin' user..."
+osc project default
```

Comment:
the switch back to cluster admin was above wait_for_build so that build-logs would work. Looks like `.Stream()` never returns an error if the request completes, regardless of http status code